<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocialNetworkWindow</class>
 <widget class="QMainWindow" name="SocialNetworkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SocialNetworkWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="loginTextEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>211</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="loginLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Your Name</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="profileTitle">
    <property name="geometry">
     <rect>
      <x>307</x>
      <y>20</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>My Profile</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="friendsTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>131</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Item</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="postsLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>331</width>
      <height>341</height>
     </rect>
    </property>
    <property name="text">
     <string>Put 5 most recent posts here</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>470</y>
      <width>251</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Go back to own profile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addFriendButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>290</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>AddFriend</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="friendSuggestionsTable">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>131</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Item</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="logoutButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>500</y>
      <width>251</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="accountCreation">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>500</y>
      <width>251</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Create new account</string>
    </property>
   </widget>
   <widget class="QLabel" name="loginLabel_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Your Name</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="loginLabel_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Your date of birth</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="loginLabel_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter your zip code</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="makeAccountButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>251</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Create New Account</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="yearSpin">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="zipSpin">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="nameBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <zorder>loginTextEdit</zorder>
   <zorder>loginButton</zorder>
   <zorder>loginLabel</zorder>
   <zorder>profileTitle</zorder>
   <zorder>friendsTable</zorder>
   <zorder>postsLabel</zorder>
   <zorder>backButton</zorder>
   <zorder>friendSuggestionsTable</zorder>
   <zorder>addFriendButton</zorder>
   <zorder>logoutButton</zorder>
   <zorder>accountCreation</zorder>
   <zorder>loginLabel_2</zorder>
   <zorder>loginLabel_3</zorder>
   <zorder>loginLabel_4</zorder>
   <zorder>makeAccountButton</zorder>
   <zorder>yearSpin</zorder>
   <zorder>zipSpin</zorder>
   <zorder>nameBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
